Makefile.dep

```diff
@@ -19,6 +19,9 @@ OLD_USEPKG := $(sort $(USEPKG))
 include $(RIOTBASE)/sys/Makefile.dep
 include $(RIOTBASE)/drivers/Makefile.dep
 
+# pull dependencies from packages
+-include $(USEPKG:%=$(RIOTPKG)/%/Makefile.dep)
```

Comment:
Forgot about this one, this change needs to be justifed.

Reply:
This is needed because the arduino dependency of the talking_leds package must be know before to effectively disable the `stdio_cdc_acm` module. I'm not sure of all the consequences of such a change but apparently Murdock doesn't complain.

Reply:
Could you have this in a separate PR, I think I now how to test the implications.